javascript/package.json

```diff
@@ -64,7 +64,7 @@
     "@types/mocha": "9.1.0",
     "@types/node": "17.0.7",
     "@types/react": "17.0.44",
-    "@types/react-dom": "17.0.14",
+    "@types/react-dom": "18.0.0",
```

Comment:
@aurelien-reeves do you know why this project has react in it?

Reply:
This is for the "try" sub-project, to try cucumber-expression inside a browser.

I guess we could move that sub-project into another location (a dedicated repo? Or simply another folder inside the current repo?)

Reply:
(Sorry for the delay in the response, I missed your message before 😓)

Reply:
https://github.com/cucumber/cucumber-expressions/issues/106